### PR TITLE
Clean up 'scripts' babushka dep

### PR DESCRIPTION
**Pull request by ags**
This is no longer used and confusing to keep around.

### CODE REVIEW

**Review by benmacleod — APPROVED** [review]
🔥 👍

**Comment by jameshill** [comment]
seriously omg. tears of joy.

**Comment by benmacleod** [comment]
Dancing around the 🔥

File: scripts/puma-statsd-logger [hunk 1]
```diff
@@ -1,76 +0,0 @@
...
-loop do
-  data = `cd #{puma_app_path} && ./bin/pumactl stats`.strip.split("\n").last
-  stats = PumaStats.new(JSON.parse(data))
-  UDPSocket.new.send("#{metric_name}.workers:#{stats.workers}|g", 0, '127.0.0.1', 8126)
-  UDPSocket.new.send("#{metric_name}.booted_workers:#{stats.booted_workers}|g", 0, '127.0.0.1', 8126)
-  UDPSocket.new.send("#{metric_name}.running:#{stats.running}|g", 0, '127.0.0.1', 8126)
-  UDPSocket.new.send("#{metric_name}.backlog:#{stats.backlog}|g", 0, '127.0.0.1', 8126)
-
-  sleep(1)
-end
```
Comment (yob) [inline scripts/puma-statsd-logger:76]: This was pretty useful back in the day! Farewell old friend 👋

Comment (jameshill) [reply scripts/puma-statsd-logger:76]: > This was pretty useful back in the day! Farewell old friend 👋

I was wondering if we were going to hear from anyone given it's a public repo

**Comment by yob** [comment]
Apparently I'm still subscribed 🤷‍♂️

**Comment by eleanorakh** [comment]
Also hello from me, who is also still subscribed! 👋🏻👋🏻👋🏻👋🏻👋🏻👋🏻

File: scripts/puma-statsd-logger (hunk 1 above)
Comment (jameshill) [reply scripts/puma-statsd-logger:76]: Just tidying up post move to ansible

**Comment by jameshill** [comment]
Oh hey there... Buildkite Party! 

statsd-logger originally inspired by Tim & Sam too!

**Pull request merged**